app/src/androidTest/java/org/simple/clinic/drugstockreminders/DrugStockReminderApiIntegrationTest.kt

```diff
@@ -0,0 +1,47 @@
+package org.simple.clinic.drugstockreminders
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.simple.clinic.TestClinicApp
+import org.simple.clinic.drugstockreminders.DrugStockReminder.Result.Found
+import org.simple.clinic.util.TestUtcClock
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+import javax.inject.Inject
+
+class DrugStockReminderApiIntegrationTest {
+
+  @Inject
+  lateinit var drugStockReminder: DrugStockReminder
+
+  @Inject
+  lateinit var clock: TestUtcClock
+
+  @Before
+  fun setUp() {
+    TestClinicApp.appComponent().inject(this)
+    clock.setDate(LocalDate.parse("2022-03-01"))
+  }
+
+  @Test
+  fun drug_stock_reports_are_filled_or_not_in_the_previous_month_should_be_fetched_to_show_reminders() {
+    // given
+    val dateFormat = DateTimeFormatter.ofPattern("YYYY-MM-DD", Locale.ENGLISH)
+    val previousMonthsDate = LocalDateTime.now(clock).minusMonths(1)
+
+    val formattedDate = dateFormat.format(previousMonthsDate)
```

Comment:
We can directly use `LocalDate#toString` which returns a string representation of a date in `yyyy-MM-dd` format (ISO-8601 format). So, we can use `LocalDate.now(clock).toString()`

It's not important to change in this PR, we can do it later as well.